### PR TITLE
raftstore: sync_log for apply snapshot

### PR DESCRIPTION
**Pull request by javaforfun**

### CODE REVIEW

**Comment by siddontang** [comment]
Ping @javaforfun

File: src/raftstore/store/peer_storage.rs [hunk 1]
```diff
@@ -1114,6 +1114,7 @@ impl PeerStorage {
                 &ready_ctx.kv_wb,
                 &ready_ctx.raft_wb
             ));
+            ready_ctx.sync_log = true;
```
[the remark this answers is not in the document]
Comment (javaforfun) [reply src/raftstore/store/peer_storage.rs]: peer_sotrage:handle_raft_ready only use writebatch,  the real write is in store:on_raft_ready

[the remark this answers is not in the document]
Comment (siddontang) [reply src/raftstore/store/peer_storage.rs]: yes, I know, what I mean here is to use sync write in the on_raft_ready directly for the KV write.

[the remark this answers is not in the document]
Comment (siddontang) [reply src/raftstore/store/peer_storage.rs]: We only need to sync KV write for the snapshot.

**Comment by siddontang** [comment]
LGTM

PTAL @BusyJay

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**